### PR TITLE
fix(TDI-34725) : change UI behavior

### PR DESCRIPTION
**Pull request by undx**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-34725 (point 2.)

**What is the new behavior?**

fixed UI behavvior

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [X] The code coverage on new code >75%
- [X] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/723/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/723/)

File: components/components-marketo/src/test/java/org/talend/components/marketo/tmarketoinput/TMarketoInputPropertiesTest.java [hunk 1]
```diff
@@ -961,4 +961,85 @@ public void testAfterFetchLeadSchema() throws Exception {
                 props.schemaInput.schema.getValue().getName());
     }
 
+    @Test
+    public void testTDI34725() throws Exception {
```
Comment (akhabali) [inline components/components-marketo/src/test/java/org/talend/components/marketo/tmarketoinput/TMarketoInputPropertiesTest.java]: Please break down this unit test and give suitable names to the test cases.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/725/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/725/)

File: components/components-marketo/src/test/java/org/talend/components/marketo/tmarketoinput/TMarketoInputPropertiesTest.java [hunk 2]
```diff
@@ -961,4 +961,85 @@ public void testAfterFetchLeadSchema() throws Exception {
                 props.schemaInput.schema.getValue().getName());
     }
 
+    @Test
+    public void testLeadActivityAndLeadChangeVisibility() throws Exception {
```
Comment (akhabali) [inline components/components-marketo/src/test/java/org/talend/components/marketo/tmarketoinput/TMarketoInputPropertiesTest.java]: you can split this test to multiple use cases, to make it easy to maintain

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/735/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/735/)

**Pull request merged**